engine/server/engine/centralized_logs/client_implementations/persistent_volume/file_layout/file_layout.go

```diff
@@ -0,0 +1,26 @@
+package file_layout
+
+import (
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_filesystem"
+	"time"
+)
+
+type LogFileLayout interface {
+	// GetLogFileLayoutFormat returns a string representation the "format" that files are laid out in
+	// Formats are composed:
+	// - "/" - representing a nested directory
+	// - "<enclaveUuid>" - representing where an enclave uuid is inserted
+	// - "<serviceUuid>" - representing where a service uuid is inserted
+	// - time formats specified by strftime https://cplusplus.com/reference/ctime/strftime/
+	// - any other ascii text
+	GetLogFileLayoutFormat() string
```

Comment:
this is the format that Vector LogsAggregator expect, probably a better way to represent this, open to idea